js/gateio.js

```diff
@@ -378,7 +381,7 @@ module.exports = class gateio extends Exchange {
         }
         return {
             'id': id,
-            'datetime': this.iso8601 (timestamp),
```

Comment:
I'm trying to understand why it did crash there, whereas it actually should've worked ok... the `iso8601` method is designed to work with undefined values, unless there was some other type of value in the `timestamp` (not undefined, but `null`, maybe?)

- https://github.com/ccxt/ccxt/blob/master/js/base/Exchange.js#L185
- https://github.com/ccxt/ccxt/blob/master/python/ccxt/base/exchange.py#L669
- https://github.com/ccxt/ccxt/blob/master/php/Exchange.php#L489